metrics-core/src/main/java/com/codahale/metrics/CsvReporter.java

```diff
@@ -220,7 +239,10 @@ private void reportTimer(long timestamp, String name, Timer timer) {
         report(timestamp,
                 name,
                 "count,max,mean,min,stddev,p50,p75,p95,p98,p99,p999,mean_rate,m1_rate,m5_rate,m15_rate,rate_unit,duration_unit",
-                "%d,%f,%f,%f,%f,%f,%f,%f,%f,%f,%f,%f,%f,%f,%f,calls/%s,%s",
+                "%d" + separator + "%f" + separator + "%f" + separator + "%f" + separator + "%f" + separator + "%f" +
```

Comment:
What do you think about using `String.join` to avoid repetition? This method uses a `StringBuilder` under the hood, so it should provide similar performance as concatenation.